tests/countries/test_ivory_coast.py

```diff
@@ -0,0 +1,184 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from unittest import TestCase
+
+from holidays.countries import IvoryCoast, CI, CIV
+from tests.common import CommonCountryTests
+
+
+class TestIvoryCoast(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(IvoryCoast, years=range(1997, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(IvoryCoast, CI, CIV)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(IvoryCoast(years=1959))
+
+    def test_2025(self):
+        self.assertHolidays(
+            IvoryCoast(years=2025),
+            ("2025-01-01", "1er janvier"),
```

Comment:
This should be placed closer to the end, after the tests of individual holidays.